aspnetcore/security/authentication/index.md

```diff
@@ -105,6 +105,15 @@ See the following links for differences between challenge and forbid:
 * [Challenge and forbid with an operational resource handler](xref:security/authorization/resourcebased#challenge-and-forbid-with-an-operational-resource-handler).
 * [Differences between challenge and forbid](xref:security/authorization/secure-data#challenge).
 
+## Authentication providers per tenant
+
+Orchard Core is:
```

Comment:
Given that `Orchard Core` is not part of the ASP.NET Core framework, I would like this section to be a bit more verbose. Here is a suggestion. Feel free to use it or tweak it as necessary:

```suggestion
ASP.NET Core framework does not have a built-in solution for multi-tenant authentication.
While it's certainly possible for customers to write one, using the built-in features, we recommend customers to look into [Orchard Core](https://www.orchardcore.net/) for this purpose.

Orchard Core is:
```